README.md

```diff
@@ -601,9 +601,7 @@ authentication
             manager.saveCredentials(credentials)
         }
     })
-```
-
-**Note:** This method is not thread-safe. Attempting to save credentials multiple times in a short period or from different threads can cause issues like invalidating valid stored tokens.  
```

Comment:
Should we instead call out that this method _is_ in fact thread safe, or is this assumed?

Reply:
I was thinking of the same. In fact, you can see the Swift [PR](https://github.com/auth0/Auth0.swift/pull/548) where they call this out.

But as a developer, I would assume the method is thread-safe unless called out hence I didn't want to overburden the documentation with these details (We have to provide version details as well). 

One of the AC for this task is to ensure we notify that we have made this method thread-safe in all the reported GH issues. I guess this should be enough. But would like to hear your POV as well if you think we need to add it.